app/models/user.rb

```diff
@@ -39,7 +39,7 @@ def self.create_on_tenant(application_instance, user)
       user_dup = User.find_or_initialize_by(
         lti_user_id: user.lti_user_id,
       )
-      user_dup.update_attributes(user.copy_attributes)
+      user_dup.update(user.copy_attributes)
```

Comment:
Should we change this to update! ? I realize the old code ignored errors but maybe it shouldn't?

Reply:
hmm, I'll leave that decision up to you. It should probably be its own PR if we make that kind of change though.